core/lib/dal/src/consensus_dal.rs

```diff
@@ -325,17 +325,11 @@ impl ConsensusDal<'_, '_> {
         &mut self,
         numbers: std::ops::Range<validator::BlockNumber>,
     ) -> DalResult<Vec<Payload>> {
-        let numbers = (|| {
-            anyhow::Ok(std::ops::Range {
-                start: L2BlockNumber(numbers.start.0.try_into().context("start")?),
-                end: L2BlockNumber(numbers.end.0.try_into().context("end")?),
-            })
-        })()
-        .map_err(|err| {
-            Instrumented::new("block_payloads")
-                .with_arg("numbers", &numbers)
-                .arg_error("numbers", err)
-        })?;
+        let i = Instrumented::new("block_payloads").with_arg("numbers", &numbers);
+        let numbers = std::ops::Range {
```

Comment:
Off-topic: Explicit `Range` construction looks weird; why not use the ordinary `start..end` syntax?

Reply:
changed